packages/web-app/src/views/components/files-cell.njk

```diff
@@ -1,17 +1,19 @@
 {% macro filesCell(data) %}
 
```

Comment:
DD is the advised pattern in gds. Why is it being changed here?

Reply:
Changed to show list of files as a list rather as in the prototype

Reply:
The prototype is just a visual representation, we ah supposed to go with gds.